app/swagger/schemas/user_internal_services.rb

```diff
@@ -45,6 +45,10 @@ class UserInternalServices
               property :gender, type: :string
               property :zip, type: :string
               property :last_signed_in, type: :string
+              property :authn_context, type: :object
```

Comment:
Do both `sign_in` and `authn_context` return a string?  If yes, you'd add this:

```ruby
property : authn_context, type: :string
property : sign_in, type: :string
```

Reply:
yes, they should both be string... in fact they're a distinct set of strings, so if we really wanted to get detailed in the schema we could include the array of possibilities, but @narin i think changing authn_context from object to string should suffice.

Reply:
@narin - FYI, for an object, you would add something like this:

```ruby
property :sign_in, type: :object do
  property :name_of_first_prop, type: :string
  property :name_of_second_prop, type: :string
end
```